.vscode/setup.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
```

Comment:
You are assuming that the correct version of Python is already installed. But the task is to write a script to provision a blank machine, so you need to install Python first.

This looks like an attempt at the "backup exercise" because you were unable to get Vagrant to work.  It lists some things to consider and "activate python" is not on the list. Neither is "install pip".
Let us know if you don't have the backup exercise pdf.